hackathon_site/dashboard/frontend/src/components/dashboard/ItemTable/ItemTable.test.tsx

```diff
@@ -73,13 +74,17 @@ describe("<PendingTable />", () => {
         const store = makeStoreWithEntities({
             pendingOrders: mockPendingOrdersInTable,
         });
-        const { getByText, queryByText } = render(<PendingTable />, { store });
```

Comment:
Why are you changing the hide button test? You should create a new test for  the cancel button.

Reply:
I wanted to add to this test for when the hide button is clicked, you wouldn't be able to see the cancel button